mesheryctl/pkg/utils/sse-client.go

```diff
@@ -43,6 +43,7 @@ func loop(reader *bufio.Reader, events chan Event) {
 			fmt.Fprintf(os.Stderr, "error during resp.Body read:%s\n", err)
 
 			close(events)
+			return
```

Comment:
Are you sure, adding return here won't affect the behavior of this at other places?

// @DelusionalOptimist